src/Acuminator/Acuminator.Utilities/Settings/VSVersion.cs

```diff
@@ -33,7 +34,9 @@ public class VSVersion
 
         public bool VS2019OrNewer => FullVersion.Major >= VS2019;
 
-        public bool IsUnknownVersion => FullVersion.Major == UnknownVersion;
+		public bool VS2022OrNewer => FullVersion.Major >= VS2022;
```

Comment:
Even this file shows that existing code uses spaces, the new formatting is broken on GitHub:
![image](https://user-images.githubusercontent.com/1544021/151838477-bc609131-0b80-4585-9aa9-6ef481dc04a8.png)

Reply:
For some weird reason my VS 2022 switched to spaces(((
Seems like weird settings sync issue - I remember setting indent settings for all languages. 
Still I would use tabs since most of the files were untouched by this PR

Reply:
fixed